src/objects/LOD.js

```diff
@@ -48,7 +48,7 @@ class LOD extends Object3D {
 
 	}
 
-	addLevel( object, distance = 0 ) {
+	addLevel( object, distance = 0, hysteresis = 0.05 ) {
```

Comment:
Revisiting this PR.

What do you think about changing the default of `hysteresis` to `0`? Having a hysteresis option is great but adding it now should probably not affect current setups. Meaning I would not "activate" it by default but let the user define an appropriate value.

Reply:
Agreed and done!